### PR TITLE
add Fran's Café

### PR DESCRIPTION
**Pull request by willemarcel**

### CODE REVIEW

**Comment by matkoniecz** [comment]
Is there a wikipedia or wikidata entry for that brand?

**Comment by willemarcel** [comment]
no

File: brands/amenity/cafe.json [hunk 1]
```diff
@@ -279,6 +279,17 @@
...
   },
+  "amenity/cafe|Fran's Café": {
+    "count": 19,
+    "countryCodes": ["br"],
+    "tags": {
+      "amenity": "cafe",
+      "brand": "Fran's Café",
+      "cuisine": "coffee_shop",
+      "name": "Fran's Café",
+      "website": "http://www.franscafe.com.br"
```
Comment (bhousel) [inline brands/amenity/cafe.json]: This should probably be `brand:website`

**Comment by bhousel** [comment]
Thanks @willemarcel - this suggestion won't make it into iD without a `brand:wikidata` tag, but it's ok to add to the index now and hope that someone comes along later and fills the entry in the rest of the way.

**Comment by ENT8R** [comment]
> Is there a wikipedia or wikidata entry for that brand?

I just created a new entry with all information I could get at the moment: https://www.wikidata.org/wiki/Q62075645

**Comment by ENT8R** [comment]
I just added the entry and corrected some things so I could merge this.
Thank you for this PR 👍

File: brands/amenity/cafe.json [hunk 2]
```diff
@@ -280,14 +280,15 @@
     }
   },
   "amenity/cafe|Fran's Café": {
-    "count": 19,
     "countryCodes": ["br"],
+    "nocount": true,
     "tags": {
       "amenity": "cafe",
       "brand": "Fran's Café",
+      "brand:website": "http://www.franscafe.com.br"
```
Comment (matkoniecz) [inline brands/amenity/cafe.json]: , is missing

**Pull request merged**